src/app/components/AllowanceMenu/index.tsx

```diff
@@ -2,6 +2,7 @@ import { GearIcon } from "@bitcoin-design/bitcoin-icons-react/filled";
 import { CrossIcon } from "@bitcoin-design/bitcoin-icons-react/outline";
 import { useState, useEffect } from "react";
 import Modal from "react-modal";
+import { useSettings } from "~/app/context/SettingsContext";
```

Comment:
Maybe we could also replace this then? Not in this but in a next PR?:
https://github.com/getAlby/lightning-browser-extension/blob/feat/738_openchannel/src/app/screens/LNURLPay.tsx/#L100-L109

Reply:
Yes! Is noted and added to the TODOs above in the description

Reply:
+ the other occurrences where we use `api.getSettings` (currencyConvert.ts, LNURLPay.tsx, utils/index.ts, batteries/index.ts, ...(?)

Reply:
The new hook `useSettings` works only on component /rendering level

I created a new ticket here as a follow up
https://github.com/getAlby/lightning-browser-extension/issues/1210